tron/mesos.py

```diff
@@ -13,10 +16,28 @@
 
 TASK_LOG_FORMAT = '%(asctime)s %(name)s %(levelname)s %(message)s'
 TASK_OUTPUT_LOGGER = 'tron.mesos.task_output'
+CLUSTERMAN_YAML_FILE_PATH = "/nail/srv/configs/clusterman.yaml"
+CLUSTERMAN_METRICS_YAML_FILE_PATH = "/nail/srv/configs/clusterman_metrics.yaml"
 
 log = logging.getLogger(__name__)
 
 
+def get_clusterman_metrics():
+    try:
+        import clusterman_metrics
+        import clusterman_metrics.util.costs
+
+        clusterman_yaml = CLUSTERMAN_YAML_FILE_PATH
+        staticconf.YamlConfiguration(
+            CLUSTERMAN_METRICS_YAML_FILE_PATH, namespace="clusterman_metrics"
+        )
```

Comment:
I am probably forgetting, but how are callers going to set this?  I am wondering if it would make sense to do something similar to what we do in spark-run where we look up the AWS region from the cluster (https://github.com/Yelp/paasta/blob/799cc1da422c6c7f53fe1d91e0eb9e0dc6d6ec3f/paasta_tools/cli/cmds/spark_run.py#L578)

Reply:
Err I am sorry, this comment is in the wrong place.